libcloud/dns/types.py

```diff
@@ -69,11 +69,9 @@ class RecordType(object):
     TXT = 'TXT'
     PTR = 'PTR'
     SOA = 'SOA'
-    HINFO = 'HINFO'
     SPF = 'SPF'
     SRV = 'SRV'
     SSHFP = 'SSHFP'
-    PTR = 'PTR'
```

Comment:
This line is duplicated twice: `PTR = 'PTR'`
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L70
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L76

This line is duplicated twice: `HINFO = 'HINFO'`
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L68
https://github.com/apache/libcloud/blob/trunk/libcloud/dns/types.py#L72